extensions/authentication-jwt/README.md

```diff
@@ -0,0 +1,59 @@
+# @loopback/extension-authentication-jwt
+
+This module exports the jwt authentication strategy and its corresponding token
+service as a component. You can mount the component to get a prototype token
+based authentication system in your LoopBack 4 application.
+
+## Usage
+
+This module contains an example LoopBack 4 application in folder `fixtures`. It
```

Comment:
As a user, I think I would be more interested to see how I can use this extension from ground zero, e.g. I have a todo application, how this extension can be used.  If User and UserCredentials are the pre-requisite for using this extension, it might be good to include there. 

I think it's better to include the models, so that users only need to have minimal setup in order to use this extension. But at the same time, we have the flexibility to extend the models for customization. Is it possible?

Reply:
@dhmlau Good question. The user service requires **User and UserCredential's persistence**. Therefore including the user service in the component means the corresponding repositories and datasource are also needed:
- User model
- UserCredential model
- User repository
- UserCredential repository
- DbDatasource

But usually a LoopBack app has its own datasource, that's why I find it hard to include the user service.

> But at the same time, we have the flexibility to extend the models for customization

It's not only about the models, but repositories and datasource are also involved...WTDY?

Reply:
Can we abstract out the implementation of the UserService and TokenService? For example, if the UserService is actually calling out other service to verify the credentials rather than validating against a database entry.

Reply:
@dhmlau I extracted the repository and model into component.
Now the app can mount the component then inject datasource and call its services in controller 👏